docs/connecting-to-a-lit-network/mainnets.md

```diff
@@ -0,0 +1,48 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Mainnets
+
+Main networks are designed for late-stage application deployment. If you are implementing storing assets with real world value, it should be done on the main networks and not the test networks. While main networks may be deprecated in the future, assets will be transferable onto new networks.
+
+
+Here is an overview of the Lit mainnets:
+
+| Name       | Lit Blockchain                                                   | Description                                                                                                                                                                         | Minimum Lit SDK Version | Lit SDK Network Identifier | Requires Payment |
+|------------|------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|----------------------------|------------------|
+| Datil-prod | [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone) | Centralized mainnet designed for production deployment. Guaranteed real world asset transferability onto new mainnets. Payment is enforced.                                                                        | `^6.4.0`                | `datil-prod`               | ✅                |
+|
+## Datil-prod
+
+The Lit network, Datil-prod, utilizes the Lit blockchain: Chronicle Yellowstone. It's a centralized mainnet designed for production deployment, and is superseding the Habanero mainnet. Like Habanero, usage of the network **does** require payment using [Capacity Credits](../sdk/capacity-credits).
+
+If you are currently on one of the Lit networks that utilize the Chronicle blockchain (i.e. Cayenne, Manzano, and/or Habanero), please refer to [this guide](./migrating-to-datil) to learn how to migrate to Chronicle Yellowstone.
```

Comment:
For this paragraph, i think we can leave out the mentions of the new rollup and frame it from the network perspective. So, for example:
> If you are currently on one of the Lit networks that utilize the Chronicle blockchain (i.e. Cayenne, Manzano, and/or Habanero), please refer to [this guide](./migrating-to-datil) to learn how to migrate to Chronicle Yellowstone.

Would become "If your application is currently deployed to a v0 Lit network (Cayenne, Manzano, and/or Habanero) please refer to [this guide] to learn how to migrate to the new Datil networks.

Reply:
Sounds good. Changed this for the `testnets` page too.